libs/front/storybook/.storybook/i18next.js

```diff
@@ -0,0 +1,40 @@
+import { initReactI18next } from 'react-i18next';
+import i18n from 'i18next';
+import LanguageDetector from 'i18next-browser-languagedetector';
+
+const ns = [
+  'common',
+  'forms',
+  'signupPage',
+  'userInfosPage'
+];
+const supportedLngs = ['en', 'fr'];
+
+i18n
+  .use(LanguageDetector)
+  .use(initReactI18next)
+  .init({
+    react: { 
+      useSuspense: false 
+    },
+    lng: 'en',
+    fallbackLng: 'en',
+    interpolation: {
+      escapeValue: false,
+    },
+    defaultNS: 'common',
+    ns,
+    supportedLngs,
+  });
+
+supportedLngs.forEach((lang) => {
+  ns.forEach((namespace) => {
+    i18n.addResourceBundle(
+      lang,
+      namespace,
+      require(`../../translations/assets/locales//${lang}/${namespace}.json`)
+    );
+  });
+});
```

Comment:
And I've done this to avoid having dozens of errors in console (for each missing translated namespace in a specific language)
```suggestion
supportedLngs.forEach((lang) => {
  let notFoundNamespacesCount = 0

  ns.forEach((n) => {
    try {
      const dir = require(`../locales/${lang}/${n}.json`)
      i18n.addResourceBundle(lang, n, dir)
    } catch (err) {
      notFoundNamespacesCount++
    }
  })

  if (notFoundNamespacesCount > 0) {
    console.warn(
      `${notFoundNamespacesCount} namespaces not found for "${lang}"`
    )
  }
})
```